lib/parse-url.js

```diff
@@ -0,0 +1,33 @@
+import {parseInput as parseEmbeds} from 'embeds';
+import objectAssign from 'object-assign';
+import isUrl from 'is-url';
+import isImageUrl from 'is-image-url';
+
+export default (elm) => {
+  if (!elm) {
+    return null;
+  }
+
+  const content = elm.textContent || '';
+  if (!isUrl(content)) {
+    return null;
+  }
+
+  if (isImageUrl(content)) {
+    return {
+      type: 'embed',
+      embedType: 'image',
+      src: content
+    };
```

Comment:
currently we host all images on our s3 - and this is something I foresee us doing in the future as well. This simply adds the image to the page - right?